event_handling/test/test_event_handling.cpp

```diff
@@ -15,7 +15,7 @@ class EventTest : public ::testing::Test
     {
         callbackInt++;
         this->cv->notify_one();
-        std::cout << "ZÄHLE HOCH " << callbackInt << std::endl;
+        std::cout << "Counting " << callbackInt << std::endl;
```

Comment:
Thanks for the translation, you just improved my non-existent german vocab!

Reply:
my pleasure